cmd/recording.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+  	"encoding/json"
+  	"net/http"
+	"os"
+
+  	"github.com/spf13/cobra"
+)
+
+var path string = "/_appmap/record"
+
+func init() {
+	var (
+		recordingCmd = &cobra.Command {
```

Comment:
This variable, and others like it, should match the subcommand being implemented (`appmapCmd`, rather than `recordingCmd`).